### ISSUE TITLE
Requesting generic Icon component to display icon using names

### ISSUE THREAD

**Issue by naseemali925** [react-icons/react-icons#392]
**Is your feature request related to a problem? Please describe.**
I'm developing a website and mobile app. For mobile, I am using react native vector icons, which gives me the flexibility to create an icon from their name after I have configured what IconSet(MaterialIcons, FontAwesome, etc) I will be using in the Provider(Some ThemeProvider). Now I can get any icon to display using their name and I won't have to manually import them all.

```jsx
import Icon from 'react-native-vector-icons/FontAwesome';
const iconName='rocket'
const myIcon = <Icon name={iconName} />;
```

**Describe the solution you'd like**
I would love a similar component that can use the name prop to display the icon.
Something like this
```jsx
import { IconContext } from "react-icons";
import Icon from 'react-icons/fa';
<IconContext.Provider
        settings={{
          icon: props => <Icon {...props} />,
        }}
        theme={this.state.theme}
      >
        // ...
</IconContext.Provider>
```

```jsx
import Icon from 'react-icons/fa';
const iconName='rocket'
const myIcon = <Icon name={iconName} />;
```

**Describe alternatives you've considered**
I really like that, this library covers most of the popular Icon Sets, and would like to use this one. 

**Additional context**

**Comment by guiguetz** [comment]
Absolutely, i came into the issues just to ask something like it. This solution seems a good approach, would use it!

**Comment by naseemali925** [comment]
> Absolutely, i came into the issues just to ask something like it. This solution seems a good approach, would use it!

Hey @guiguetz , the solution I suggested has not yet been implemented by the team at react-icons. How do you plan to use it?

**Comment by leonardomdr** [comment]
I would love to have an option for this as well...
